scripts/.eslintrc.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+# Necessary to support async/await... grumble grumble...
+parserOptions:
+  ecmaVersion: 8
```

Comment:
You can do this with a eslint comment within the `changelog.js` file instead of making a config file.

Reply:
It seems to me that `parserOptions` configs cannot be part of an ESLint comment 💩
Or at least I couldn't find how... Any idea?